### ISSUE TITLE
Do not able to learn the latent representations

### ISSUE THREAD

**Issue by dattnguyenx** [fab-jul/L3C-PyTorch#35]
Thank you for the excellent work. 
In your paper, you stated that "by stopping gradients from propagating through the targets of our loss, we get significantly worse performance – in fact, the optimizer does not manage to pull down the cross-entropy of any of the learned representations z(s) significantly".
What do you mean by stopping gradients? do you have any method to force propagate the gradients? 
I build my own model, the network is able to learn the RGB scale but has high losses on lower scales.
